packages/hash/external-services/kratos/kratos.dev.yml

```diff
@@ -23,6 +23,18 @@ selfservice:
     password:
       enabled: true
 
+    link:
+      config:
+        # The URL for verification emails are set through the link method
+        # but we're using the code method, so we disable this method for usage.
+        enabled: false
+        # Set through SELFSERVICE_METHODS_LINK_CONFIG_BASE_URL
+        base_url: http://localhost:3000/api/ory
+    code:
+      config:
+        # and make sure to enable the code method.
+        enabled: true
```

Comment:
Nice, should we make an effort from here on to start copying in the documentation from the ory kratos website?

Reply:
I think it's very helpful for values that we know we will inject new values into such as this and the `UI_URL`s. Fortunately their env naming matches the path of the `yaml` but in upper case and with underscores. I think it's helpful to have the env var to be easily-copied, though